sys/net/gnrc/sock/include/sock_types.h

```diff
@@ -25,6 +25,7 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+#include "event.h"
```

Comment:
This probably should be: 

```c
#ifdef SOCK_HAS_ASYNC
#include "event.h"
#endif
```

Reply:
Doesn't hurt, though it is not necessary